pom.xml

```diff
@@ -102,6 +102,8 @@
     <spark.version>1.6.1</spark.version>
     <scala.version>2.10.4</scala.version>
     <scala.binary.version>2.10</scala.binary.version>
+    <springboot.version>1.3.6.RELEASE</springboot.version>
+    <zipkin.version>1.5.1</zipkin.version>
```

Comment:
Similarly, Zipkin has a 1.9.0 release they just put out. Is there a reason for using 1.5.1?